api/src/about/about.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { AboutService } from './about.service';
+import { About } from './entity/about.entity';
+
+@Controller('about')
+export class AboutController {
+  constructor(private readonly aboutService: AboutService) {}
+
+  @Get()
+  async findAll(): Promise<About[]> {
+    return this.aboutService.findAll();
+  }
+
+  @Get(':id')
```

Comment:
based on the re-modeling comment, I think you should remove this as this will be single resource.